src/main/java/com/jalasoft/webservice/entitities/TextFile.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ *  This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
+package com.jalasoft.webservice.entitities;
+
+import com.jalasoft.webservice.entitities.ModelFile;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+/** TextFile : Class to manipulate information related to CSV files
+ * Version : 1.0
+ * Date: 9/19/2019
+ */
+
+public class TextFile extends ModelFile {
+    private String text;
+
+    public String getText() {
+        return text;
+    }
+
+    public void setText(String text) {
+        this.text = text;
+    }
+
+    /***
+     * Method to save text in a file
+     * @param text
+     */
+    public void saveTextIntoFile(String text) {
+        String csvFile = getPath() + getFullFileName();
+        BufferedWriter bufferWriter = null;
+        FileWriter fileWriter = null;
+        setText(text);
+        try {
+
+            fileWriter = new FileWriter(csvFile);
+            bufferWriter = new BufferedWriter(fileWriter);
+
+            bufferWriter.write(text);
+
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
use logger print the exception instead of `e.printStackTrace();`

``` java
 LOGGER.info("Exception writing file:", e);
```
before this you need to declare LOGGER like:

```java
public class TextFile extends ModelFile {
    private static final Logger LOGGER = LogManager.getLogger();
```

Reply:
the same for below  ex.printStackTrace();

Reply:
done